scripts/test_device_detector_base.py

```
@@ -0,0 +1,128 @@
+    print("\n=== Environment Information ===")
+    print(f"Python: {sys.version}")
+    print(f"Platform: {sys.platform}")
+    
+    # Check if running in container
+    try:
+        with open('/proc/1/cgroup', 'r') as f:
+            if 'docker' in f.read():
+                print("Running in: Docker container")
+            else:
+                print("Running in: Bare metal")
+    except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)